core/trino-main/src/test/java/io/trino/execution/scheduler/TestSourcePartitionedScheduler.java

```diff
@@ -667,7 +667,7 @@ private StageScheduler getSourcePartitionedScheduler(
                 () -> false);
     }
 
-    private static PlanFragment createFragment()
+    public static PlanFragment createFragment()
```

Comment:
Don't expose this method, instead copy this method to `TestScaledWriterScheduler`. This will keep tests independent to each other.

Reply:
sure.